Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb

```diff
@@ -0,0 +1,32 @@
+#Enter the number below which you need to find prime numbers
+#In this case 2..30
+
+n = 2..30
```

Comment:
If it possible to take dynamic inputs in Ruby, then please take the input n for Sieve dynamically.